olive/model/__init__.py

```diff
@@ -153,6 +155,7 @@ def to_json(self, check_object: bool = False):
                 for resource_name, resource_path in self.resource_paths.items()
             },
         }
+        config["config"].update({"model_attributes": self.model_attributes})
```

Comment:
I think it's cleaner if we add it directly after line 155?

Reply:
Nope? The above config is got from python list expression. We cannot append it into a list expression.

Reply:
Or, do you mean other ways?

Reply:
Oh you are right. Thanks!